src/test/java/core/basesyntax/model/dao/FruitDaoImplTest.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax.model.dao;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.dto.Fruit;
+import core.basesyntax.model.dto.FruitRecordDto;
+import core.basesyntax.storage.FruitStorage;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FruitDaoImplTest {
+    private final FruitDao fruitDao = new FruitDaoImpl();
```

Comment:
let's initialize the fields in the method beforeClass

Reply:
and in another classes also

Reply:
@boroda4436 told this initialisation should be fine in one of our lectures.

Reply:
ok)